AsyncDisplayKit/ASMapNode.h

```diff
@@ -12,9 +12,9 @@
 @interface ASMapNode : ASImageNode
 
 /**
- The current region of ASMapNode. This can be set at any time and ASMapNode will animate the change. This property may be set from a background thread before the node is loaded, and will automatically be applied to define the region of the static snapshot (if .liveMap = NO) or the internal MKMapView (otherwise).
+ The current options of ASMapNode. This can be set at any time and ASMapNode will animate the change. This property may be set from a background thread before the node is loaded, and will automatically be applied to define the behavior of the static snapshot (if .liveMap = NO) or the internal MKMapView (otherwise).
  */
-@property (nonatomic, assign) MKCoordinateRegion region;
+@property (nonatomic, readwrite) MKMapSnapshotOptions *options;
```

Comment:
Ha! I was going against my gut here, opted for `strong` first but then seen other parts of the framework use `readwrite` so went for that. No problem, will fix that.

In terms of animating the properties that can be animated, will be. This animation obviously only occurs for the liveMap so will make these comments clearer, also highlighting which properties will animate.